tests/unit/test_client_logging.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import pytest
+
+from google.api_core.client_logging import setup_logging
```

Comment:
Another testcase would be this sequence:
1. Mock so that `os.getenv` returns `google.foo`
2. `initialize_logging()` 
3. check configs are what we expect for `google` and `google.foo`
4. Change the logger configs for  BOTH `google.foo` and `google`
5. Call `initialize_logging()` again
6. Check that the changes you made in step 4 are still there

Reply:
addressed.